library/src/androidTest/java/org/xmtp/android/library/TestHelpers.kt

```diff
@@ -41,6 +54,82 @@ class FakeWallet : SigningKey {
         get() = privateKey.walletAddress
 }
 
+class FakeSCWWallet : SigningKey {
+    private var web3j: Web3j = Web3j.build(HttpService("http://10.0.2.2:8545"))
+    private val credentials: Credentials =
```

Comment:
The credentials are a little confusing here. They get used to deploy the SCW factory, deploy the smart contract wallet, and they are the wallet that signs messages.

The first two need to be funded, so they kinda have to be the default anvil keys. The last one can be anything so you can create multiple SCWs in tests and have them talk to each other.

Maybe we can break this up to `contractDeployerCredentials` and `ownerCredentials` or something along those lines.